docker.nix

```diff
@@ -125,8 +129,12 @@ let
     sandbox = "false";
     build-users-group = "nixbld";
     trusted-public-keys = "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=";
-  };
-  nixConfContents = (lib.concatStringsSep "\n" (lib.mapAttrsFlatten (n: v: "${n} = ${v}") nixConf)) + "\n";
+  } // (if substituters == [] then {} else {
```

Comment:
I think it would be better to extend `nix.conf` via some more general `extraNixConf` variable (as simple string) instead of having specialized arguments for each option in `nix.conf`.